environment.yml

```diff
@@ -20,6 +20,7 @@ dependencies:
 - setuptools
 - python-build
 - boa
+- pycifrw
```

Comment:
where is this used? ctrl+f doesnt reveal anything

Reply:
This is needed in order for LoadCIF to work.  See output on failing test here: https://github.com/neutrons/SNAPRed/actions/runs/4995107184/jobs/8946555614#step:6:161

Reply:
sounds good